package.json

```diff
@@ -68,7 +68,8 @@
     "form-data": "0.1.4",
     "fs-extra": "0.30.0",
     "infinite-loop-loader": "1.0.1",
-    "jade": "1.11.0",
+    "jstransformer-clean-css": "^2.0.0",
+    "jstransformer-uglify-js": "^1.2.0",
```

Comment:
As I don't know much about those libraries, where/how do those helpers being registered/used?  (I guess you using the clean-css [here](https://github.com/opentable/oc/pull/497/files#diff-52280d37108c931558964db275372680R18), but couldn't find other references to those in the PR)